lib/sse.ts

```diff
@@ -44,7 +47,7 @@ export function sse<
         }
       });
 
-      let result = yield* op(events.send);
+      let result = yield* op(request, events.send);
```

Comment:
What about using `SSEHandlerOptions` here. That way, you don't have to create a throw-away alias for either `_request` or `_send` depending on which one you need.

```ts
interface SSEHandlerOptions<T> {
  request: Request;
  send(value: T): Operation<void>;
}
```

That would let us de-structure the options to get whichever one we want:

```ts
sse(function*({ request, send }) {
/*...*/
});
```